dev/SapphireTest.php

```diff
@@ -221,7 +221,7 @@ public function setUp() {
 		// Set up email
 		$this->originalMailer = Email::mailer();
```

Comment:
If unit tests relied on $this->originalMailer then they would fail if this was removed.